Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -577,6 +589,7 @@ def resend_first_contact(email_selected_thread, email_thread, incident_id, new_e
         service_mail: Address the email is sent from
         mail_sender_instance: The service email (sender address)
         new_attachment_names: List of attachment file names
+        subject_include_incident_id: Should we include the incident id in the email subject.
```

Comment:
Why is it a parameter?
Why not always include the `incident id` in the email subject?